Utils/github_workflow_scripts/handle_external_pr.py

```diff
@@ -180,6 +192,20 @@ def is_requires_security_reviewer(pr_files: list[str]) -> bool:
     return False
 
 
+def is_tim_reviewer_needed(packs_in_pr: set[str]) -> bool:
+    for pack in packs_in_pr:
+        pack_metadata = get_pack_metadata(pack)
+        for pr_file in pack:
```

Comment:
From your code here it looks like the `pack` contains a list of files but according to `packs_to_check_in_pr` which outputs the list of `packs_in_pr` it only contains a list of strings.

Reply:
Yes, I did several changes to this function and forgot the pr_files argument - fixed